src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -195,11 +199,11 @@
                 //multiple instances - until multiple are accepted, send the selected instance value for instance with
                 //greatest number of options. See VIMC-4561.
                 let instances = {};
-                if (this.metadata.instances_supported && this.metadata.instances &&
-                    Object.keys(this.metadata.instances).length > 0) {
-                    const instanceName = Object.keys(this.metadata.instances).sort((a, b) => this.metadata.instances[b].length - this.metadata.instances[a].length)[0];
+                if (this.metadata?.instances_supported && this.metadata?.instances &&
+                    Object.keys(this.metadata?.instances).length > 0) {
+                    const instanceName = Object.keys(this.metadata?.instances).sort((a, b) => this.metadata?.instances[b].length - this.metadata.instances[a].length)[0];
                     const instance = this.selectedInstances[instanceName];
-                    instances = Object.keys(this.metadata.instances).reduce((a, e) => ({[e]: instance, ...a}), {});
+                    instances = Object.keys(this.metadata?.instances).reduce((a, e) => ({[e]: instance, ...a}), {});
```

Comment:
```suggestion
                    instances = Object.keys(this.metadata!.instances).reduce((a, e) => ({[e]: instance, ...a}), {});
```

Reply:
Have made this is a little clearer now hopefully but adding an explicit null check for `this.metadata` to the initial condition